src/main/kotlin/Novu.kt

```diff
@@ -62,36 +62,36 @@ class Novu(
     fun trigger(body: TriggerEventRequest) = runBlocking {
         val response = eventsApi.triggerEvent(body)
         if (response.isSuccessful) {
-            response.body().apply { logger.info { this } }
+            response.body().apply { logger.debug { this } }
```

Comment:
This looks great but it is repeated everywhere. I think we should make this a function so we do not repeat it everywhere.

Reply:
It's just a one-liner. If it's extracted into a function, the function will also be called everywhere. There's really no difference